src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Naming convention'
+order: 5
+---
+
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
+
+We chose to use english in all our files to be understandable by the most. We also chose to use the PascalCase standard in our prototyping tools.
+
+## File Naming Conventions
+
+We use two letters prefixes to name our files, this two letters depending on the pattern type and look like that: **_[XX] File name.sketch_**
+
+| Prefix   | Pattern type        | Example                                     |
+| -------- | ------------------- | ------------------------------------------- |
+| **[AA]** | Styles              | Colors, icons, fonts, shadows, ...          |
+| **[BA]** | Simple Components   | Buttons, links, tags, forms, ...            |
+| **[BB]** | Complex Components  | Notifications, accordions, popins, tabs ... |
```

Comment:
The examples seems to be arbitrary. 
To me, the definition of a complex component could be components that uses simple components as building blocks. A composition of simple components